src/dbos.ts

```diff
@@ -251,17 +271,27 @@ export class DBOS {
     if (!DBOS.dbosConfig) {
       const [dbosConfig, runtimeConfig] = parseConfigFile({ forceConsole: isDebugging });
       if (!isDebugging) {
-        dbosConfig.poolConfig = await db_wizard(dbosConfig.poolConfig);
+        dbosConfig.poolConfig = await db_wizard(dbosConfig.poolConfig!);
       }
       DBOS.dbosConfig = dbosConfig;
       DBOS.runtimeConfig = runtimeConfig;
+    } else if (!isDeprecatedDBOSConfig(DBOS.dbosConfig)) {
+      DBOS.translateConfig();
+      if (!isDebugging && DBOS.dbosConfig.poolConfig) {
```

Comment:
How could `poolConfig` not be set here? I'd rather do the explicit type assertion (`!`) than this kind of fake check that makes the type checker happy but really will fail catastrophically if the check isn't true.

Reply:
it shouldn't be unset -- I removed the check. But why you'd it fail catastrophically? It'd just skip running the wizard and fail a bit later, before launch() exits (compared to failing the same way in the wizard)